src/adapter/src/command.rs

```diff
@@ -428,188 +380,15 @@ impl ExecuteResponse {
             }
             Prepare => Some("PREPARE".into()),
             SendingRows { .. } => None,
-            SetVariable { tag, .. } => Some(tag.to_string()),
+            SetVariable { reset: true, .. } => Some("RESET".into()),
+            SetVariable { reset: false, .. } => Some("RSET".into()),
```

Comment:
I believe you if you say this is the expected output, but is this a typo?

Reply:
Oops, yes, thanks! Fixed.